config/content.json

```diff
@@ -21,7 +21,7 @@
             "/docs/autoscale/v1/developer-guide/": "https://github.com/rackerlabs/otter/",
             "/docs/cloud-servers/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-servers/",
             "/docs/cloud-files/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-files/",
-            "/docs/metrics/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-metrics/",
+            "/docs/metrics/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-metrics/",
```

Comment:
Yes. Because docs.rackspace.com is going to the v1 Uri.  Is the redirect something that I can do, or is that you?